enrolment-ui/src/index.js

```diff
@@ -24,7 +24,7 @@ import {
 } from './modules';
 
 // Service Worker for PWA
```

Comment:
```suggestion
// Removing a previously existing Service Worker, added by the original `Create React App` to allow PWA,
// because it is creating caching issues for some users. Will remove this in a few cycles, to allow propagation.
```